app/src/main/java/com/mymeetings/android/services/LocalCalendarChangeEventReceiver.kt

```diff
@@ -9,13 +9,14 @@ import com.mymeetings.android.model.strategies.CalendarFetchStrategyType
 import org.koin.standalone.KoinComponent
 import org.koin.standalone.get
 
-class LocalCalendarChangeEventReceiver()
+class LocalCalendarChangeEventReceiver
     : BroadcastReceiver(), KoinComponent {
 
     private val calendarEventsSyncManager: CalendarEventsSyncManager = get()
+    private val consoleLog : ConsoleLog = get()
 
     override fun onReceive(context: Context?, intent: Intent?) {
-        ConsoleLog.i(message = "Receiver received")
+        consoleLog.i(message = "on received called")
```

Comment:
Should the log message be _onReceive called_?

Reply:
Changed